src/AutoRest.CSharp.V3/Generation/Writers/RestClientWriter.cs

```diff
@@ -191,6 +191,43 @@ private void WriteRequestCreation(CodeWriter writer, RestClientMethod clientMeth
                             writer.Line($");");
                         }
                         break;
+                    case MultipartRequestBody multipartRequestBody:
+                        var multipartContent = new CodeWriterDeclaration("content");
+                        writer.Line($"var {multipartContent:D} = new {typeof(MultipartFormDataContent)}();");
+
+                        foreach (var bodyParameter in multipartRequestBody.Value)
+                        {
+                            using (WriteValueNullCheck(writer, bodyParameter))
+                            {
+                                if (bodyParameter.Type.Name == "IEnumerable")
```

Comment:
Where is this logic used?

Reply:
https://github.com/Azure/autorest.csharp/pull/886#discussion_r483856367